tools/jdk/BUILD

```diff
@@ -129,8 +129,12 @@ alias(
 filegroup(
     name = "ijar",
     srcs = select({
-        "//src/conditions:remote": ["//third_party/ijar:ijar"],
-        "//conditions:default": glob(["ijar/*"]),
+        # The ijar cc_binary target.
+        "//src/conditions:remote": ["@remote_java_tools//:ijar"],
+        # TODO(#6316): Move ijar.exe to a Windows specific Java tools archive.
+        "//src/conditions:windows": ["ijar/ijar.exe"],
+        # The pre-built binary.
+        "//conditions:default": ["@remote_java_tools//:ijar_binary"],
```

Comment:
Before I take a closer look: how are you planning to deal with the fact that you need to compile ijar for multiple platforms? Before, the Java tools repository was platform-independent and the moment you put ijar into it, you'll need to have a separate one for each platform.